scss/civicrm/administer/financial/_financial-account.scss

```diff
@@ -15,4 +17,11 @@
       margin-bottom: 0;
     }
   }
+
+  #{civi-dialog()} {
+    .form-item {
+      @include civicrm-table;
+      margin: 0 -20px;
```

Comment:
if this is what happens with any table inside a dialog, shouldn't this be global

Reply:
No not with all tables, some table can be form as well.